bpython/curtsiesfrontend/repl.py

```diff
@@ -1811,7 +1816,15 @@ def prompt_for_undo():
 
         greenlet.greenlet(prompt_for_undo).switch()
 
-    def reevaluate(self, insert_into_history=False):
+    def prompt_redo(self):
```

Comment:
I think just redo, "prompt_undo" is because it may prompt you for how many lines you want to undo.